homeassistant/components/foscam/camera.py

```diff
@@ -145,25 +153,20 @@ async def async_added_to_hass(self):
             self._foscam_session.get_motion_detect_config
         )
 
-        if ret != 0:
+        if ret == -3:
+            LOGGER.info(
+                "Can't get motion detection status, camera %s configured with non-admin user.",
```

Comment:
We don't end logging messages with period.

Reply:
Fixed